nginx.conf

```diff
@@ -58,43 +58,36 @@ http {
-        "~^/chainguard/chainguard-enforce/reference/configure-enforcer-options-during-installation(.+)?$" /chainguard/chainguard-enforce/installation/configure-enforcer-options-during-installation$1;
-        "~^/chainguard/chainguard-images/reference/sdk/(.+)?$"  /chainguard/chainguard-images/reference/;
-        "~^/chainguard/chainguard-images/reference/musl-dynamic/(.+)?$"  /chainguard/chainguard-images/reference/;
-        "~^/chainguard/chainguard-images/reference/nri-kube-events/(.+)?$"  /chainguard/chainguard-images/reference/;
-        "~^/chainguard/chainguard-images/reference/nri-kubernetes/(.+)?$"  /chainguard/chainguard-images/reference/;
-        "~^/chainguard/chainguard-images/reference/nri-prometheus/(.+)?$"  /chainguard/chainguard-images/reference/;
-        "~^/chainguard/chainguard-images/reference/gcc-musl/(.+)?$"  /chainguard/chainguard-images/reference/;
-        "~^/chainguard/chainguard-images/reference/flux-reflector-controller/(.+)?$"  /chainguard/chainguard-images/reference/;
-        "~^/chainguard/chainguard-images/reference/helm-controller/(.+)?$"  /chainguard/chainguard-images/reference/;
-        "~^/chainguard/chainguard-images/reference/hubble-ui-backend/(.+)?$"  /chainguard/chainguard-images/reference/;
-        "~^/chainguard/chainguard-images/reference/k3s-embedded/(.+)?$"  /chainguard/chainguard-images/reference/;
-        "~^/chainguard/chainguard-images/reference/notification-controller/(.+)?$"  /chainguard/chainguard-images/reference/;
-        "~^/chainguard/chainguard-images/reference/source-controller/(.+)?$"  /chainguard/chainguard-images/reference/;
-        "~^/chainguard/chainguard-images/reference/dreamfactory/(.+)?$"  /chainguard/chainguard-images/reference/;
-        "~^/chainguard/chainguard-enforce/(.+)?$"  /;
+        # reference docs redirects - images
+        "~^/chainguard/chainguard-images/reference/(.+)/image_specs/?$" https://images.chainguard.dev/directory/image/$1/specifications;
+        "~^/chainguard/chainguard-images/reference/(.+)/tags_history/?$" https://images.chainguard.dev/directory/image/$1/versions;
+        "~^/chainguard/chainguard-images/reference/(.+)/provenance_info/?$" https://images.chainguard.dev/directory/image/$1/provenance;
+        "~^/chainguard/chainguard-images/reference/(.+)/overview/?$" https://images.chainguard.dev/directory/image/$1/overview;
+        "~^/chainguard/chainguard-images/reference/(.+)/$" https://images.chainguard.dev/directory/image/$1/overview;
+        "~^/chainguard/chainguard-images/reference/(.+)$" https://images.chainguard.dev/directory/image/$1/overview;
+        "~^/chainguard/chainguard-images/reference/?$" https://images.chainguard.dev/directory;
     }
 
     server {
         listen       8080;
         server_name  localhost;
         root   /usr/share/nginx/html/;
 
+        # process $request_uri -> $redirect_url if url is absolute
+        if ($redirect_url ~ ^https://) {
+            rewrite ^(.*)$ $redirect_url permanent;
```

Comment:
thanks for the review @erikaheidi !
Just FYI this is ready to merge but I am OOO Jul 15-19, idk how releases work for edu but depending on our level of comfort here we can merge and deploy now or wait til I'm back, up to you?  I expect this to be a binary everything works or everything explodes and having checked in staging with Jamon, it should be the former?

Reply:
I believe we're good to go ahead, so I will merge. If something explodes, which I don't expect, we can revert it. Thanks @julienv3 !